packages/contracts/test/BorrowerOperationsTest.js

```diff
@@ -1017,7 +1033,7 @@ contract('BorrowerOperations', async accounts => {
 
       // Check ZERO ZUSD balance after has increased
       const zeroStaking_ZUSDBalance_After = await zusdToken.balanceOf(zeroStaking.address)
-      assert.isTrue(zeroStaking_ZUSDBalance_After.gt(zeroStaking_ZUSDBalance_Before))
+      assert.isTrue(zeroStaking_ZUSDBalance_After.eq(zeroStaking_ZUSDBalance_Before))
```

Comment:
why have you changed from `gt` to `eq` ... :thinking:

Reply:
ZeroStaking doen't receive ZUSD fees anymore, all the fees are sent to SOVFeeCollector